test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -296,6 +296,46 @@ function ExecuteWebRequest
     return $result
 }
 
+# Starts the ClientCertificateCheck listener
+function StartClientCertificateCheck 
```

Comment:
@iSazonov I tried to match the other private functions in the test already. I originally wanted to use `Start-ClientCertificateCheck`, but though I would get a note for not matching the others. I can't win  :D